server/routes/charts.js

```diff
@@ -126,5 +144,49 @@ router
       return res.status(500).json({ message: error.message })
     }
   })
+  .get(ensureAuthenticated, async (req, res) => {
+    try {
+      const { dataDb } = req.app.locals
+      const chartForm = await dataDb
+        .collection(collections.charts)
+        .findOne({ _id: new ObjectID(req.params.chart_id) })
+      return res.status(200).json({ data: chartForm })
+    } catch (error) {
+      console.error(error)
 
+      return res.status(500).json({ message: error.message })
+    }
+  })
+  .put(ensureAuthenticated, async (req, res) => {
+    try {
+      const { dataDb } = req.app.locals
+      const { chart_id } = req.params
+      const { title, variable, assessment, description, fieldLabelValueMap } =
+        req.body
+      const { value, ok } = await dataDb
+        .collection(collections.charts)
+        .findOneAndUpdate(
+          { _id: new ObjectID(chart_id) },
+          {
+            $set: {
+              title,
+              variable,
+              assessment,
+              description,
+              fieldLabelValueMap,
+              updatedAt: new Date().toISOString(),
+            },
+          },
+          {
+            upsert: true,
```

Comment:
do we want to upsert or just update? IMO we don't want to insert if we don't have a chart already.